defaults/main.yml

```diff
@@ -35,3 +35,10 @@ LOAD_BALANCER_SNAKE_OIL_CERT: ssl-cert-snakeoil.pem
 
 # The interval between invocations of manage_certs.py in minutes.
 LOAD_BALANCER_MANAGE_CERTS_INTERVAL: 5
+
+# Configuration for Nginx server serving maintenance pages.
+MAINTENANCE_SERVER_PORT: 8888
+MAINTENANCE_SCHEDULED_SERVER_PORT: "{{ MAINTENANCE_SERVER_PORT + 1 }}"
+MAINTENANCE_UNSCHEDULED_SERVER_PORT: "{{ MAINTENANCE_SERVER_PORT + 2 }}"
+MAINTENANCE_SERVER_STATIC_ROOT: /srv/maintenance/
```

Comment:
Is `/srv/` used anywhere else? Seems kind of random compared to `/var/www/maintenance` or similar. I'm not objecting though, just curious.

Reply:
I use SuSE so I'm more familiar with using that location for web data. It also matches up with FHS. It's better for things to be in a place where people can know where they are without needing to read the configuration or docs though. So I'll move it to ``/var/www/maintenance``.